test/unit-test/test_mgmt_archive_support.py

```diff
@@ -208,6 +324,89 @@ def test_fetch_module_repo_from_s3(
         assert os.path.exists(os.path.join(archive_path, module_name, "modulestack.yaml"))
 
 
+@pytest.mark.mgmt
+@pytest.mark.mgmt_archive_support
+@pytest.mark.parametrize(
+    "s3_bucket_http_url", ["testing-bucket.s3.amazonaws.com", "testing-bucket.s3.us-west-2.amazonaws.com"]
+)
+def test_fetch_module_repo_from_s3_non_nested(
+    session_manager: None, archive_file_data_not_nested: Tuple[bytes, str], s3_bucket_http_url: str
+) -> None:
+    archive_bytes, archive_extension = archive_file_data_not_nested
+
+    response_mock = MagicMock()
+    response_mock.status_code = 200
+    response_mock.content = archive_bytes
+
+    s3_http_url = f"https://{s3_bucket_http_url}/testing-modules.{archive_extension}"
+    module_name = "./"
+
+    with patch("requests.get", return_value=response_mock) as mock_requests_get:
+        archive_path_test = f"archive::{s3_http_url}?module={module_name}"
+        archive_path, subdir = archive.fetch_archived_module(release_path=archive_path_test)
```

Comment:
Should we have any asserts against `subdir`?

Reply:
Since we are testing at the `fetch_archived_module` method, we are getting the full extracted path returned (no access to the actual subdir object)...but I can look at asserting the full path